src/test/groovy/com/stormpath/tck/util/ApiKey.groovy

```diff
@@ -29,22 +29,29 @@ class ApiKey {
     public static final String DEFAULT_API_KEY_PROPERTIES_FILE_LOCATION =
             System.getProperty("user.home") + File.separatorChar + ".stormpath" + File.separatorChar + "apiKey.properties";
 
-    private final static String apiKeyIdPropertyName     = "apiKey.id"
-    private final static String apiKeySecretPropertyName = "apiKey.secret"
+    private final static String API_KEY_ID_PROPERTY_NAME = "apiKey.id"
+    private final static String API_KEY_SECRET_PROPERTY_NAME = "apiKey.secret"
 
-    private final static String[] apiKeyIdEnvironmentVariableNames = ["STORMPATH_API_KEY_ID", "STORMPATH_CLIENT_APIKEY_ID"]
-    private final static String[] apiKeySecretEnvironmentVariableNames = ["STORMPATH_API_KEY_SECRET", "STORMPATH_CLIENT_APIKEY_SECRET"]
+    private final static String API_KEY_FILE_ENV_NAME = "STORMPATH_API_KEY_FILE";
+
+    private final static String[] API_KEY_ID_ENVIRONMENT_VARIABLE_NAMES = ["STORMPATH_API_KEY_ID", "STORMPATH_CLIENT_APIKEY_ID"]
+    private final static String[] API_KEY_SECRET_ENVIRONMENT_VARIABLE_NAMES = ["STORMPATH_API_KEY_SECRET", "STORMPATH_CLIENT_APIKEY_SECRET"]
 
    static {
         //1. Try to load the default api key properties file.  All other config options have higher priority than this:
         Properties properties = getDefaultApiKeyFileProperties()
-        apiKeyID = properties.get(apiKeyIdPropertyName)
-        apiKeySecret = properties.get(apiKeySecretPropertyName)
+        apiKeyID = properties.get(API_KEY_ID_PROPERTY_NAME)
+        apiKeySecret = properties.get(API_KEY_SECRET_PROPERTY_NAME)
+
+        //2. Try api key properties file environment variable
+       properties = getApiKeyFilePropertiesByEnv();
+       apiKeyID = properties.get(API_KEY_ID_PROPERTY_NAME)
+       apiKeySecret = properties.get(API_KEY_SECRET_PROPERTY_NAME)
 
-        //2. Try environment variables:
+        //3. Try environment variables:
         properties = getEnvironmentVariableProperties();
-        apiKeyID = properties.get(apiKeyIdPropertyName) != null ? properties.get(apiKeyIdPropertyName) : apiKeyID
-        apiKeySecret = properties.get(apiKeySecretPropertyName) != null ? properties.get(apiKeySecretPropertyName) : apiKeySecret
+        apiKeyID = properties.get(API_KEY_ID_PROPERTY_NAME) != null ? properties.get(API_KEY_ID_PROPERTY_NAME) : apiKeyID
```

Comment:
Not required to merge the PR, but for the future, you can use groovy's elvis operator (to be idiomatic):

    apiKeyId = properties.get(API_KEY_ID_PROPERTY_NAME) ?: apiKeyId